recipes/tandemtwister/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "tandemtwister" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Lionward/tandemtwister.git
+  git_tag: v0.1.0
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  run_exports:
+    - {{ pin_subpackage('tandemtwister', max_pin="x.x") }}
+
+requirements:
+  build:
+  - {{ compiler('cxx') }}
+  - make 4.4.1
+  host:
+    - libdeflate >=1.22,<1.26
+    - htslib ==1.22.1
+    - mlpack ==4.5.0
+    - cereal ==1.3.2
+    - spdlog ==1.15.3
+    - fmt ==11.2.0
+    - libgomp  # [linux]
+  run:
+    - htslib ==1.22.1
+    - mlpack ==4.5.0
+    - fmt ==11.2.0
+    - libgomp
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/tandemtwister
+    - tandemtwister --help 2>&1 | head -n 1
+
+about:
+  home: https://github.com/Lionward/tandemtwister
+  license: BSD-3-Clause-Non-Commercial
+  license_family: BSD
+  license_file: LICENSE
+  summary: Detection of interleaved and embedded tandem repeats from long reads
+  description: |
+    TandemTwister is a tool designed for the detection of interleaved and embedded
+    tandem repeats from long reads, facilitating advanced genomic analysis.
+    This software is licensed for NON-COMMERCIAL USE ONLY under the BSD 3-Clause
+    Non-Commercial License. For commercial licensing inquiries, please contact
+    the author.
+  doc_url: https://github.com/Lionward/tandemtwister
+
+
+extra:
+  recipe-maintainers:
+    - lionward
+  skip-lints:
```

Comment:
I think these skip-lints can be removed: the first two when the source URL is changed to tag archive tarball and its checksum is supplied, while `should_be_noarch_generic` shouldn't apply anyway (I'm guessing this was added when the whole build was being skipped by mistake).